src/odemis/gui/cont/export.py

```diff
@@ -168,15 +176,62 @@ def export_viewport(self, filepath, export_format, export_type):
             self._conf.last_export_path = os.path.dirname(filepath)
 
             exported_data = self.export(export_type, raw)
-            # record everything to a file
-            exporter.export(filepath, exported_data)
 
-            popup.show_message(self._main_frame,
-                               "Exported in %s" % (filepath,),
-                               timeout=3
-                               )
-
-            logging.info("Exported a %s view into file '%s'.", export_type, filepath)
+            # batch export
+            # TODO: for now we do not create a new folder where all files are saved
+
+            if isinstance(exported_data, dict):
+                # get the file names
+                filename_dict = {}
+                for key in exported_data.keys():
+                    base = os.path.basename(filepath)
+                    try:  # .png
+                        base_name = os.path.splitext(os.path.splitext(base)[0])[0]
+                        file_extension = os.path.splitext(os.path.splitext(base)[0])[1] + os.path.splitext(base)[1]
+                    except AttributeError:  # .ome.tiff
```

Comment:
It should be possible to replace these 6 lines by `base_name, file_extension = util.dataio.splitext(base)`